_posts/en/posts/2016-01-26-segwit-benefits.md

```diff
@@ -5,11 +5,24 @@ id: en-segwit-benefits
 title: Segregated Witness Benefits
 permalink: /en/2016/01/26/segwit-benefits
 version: 1
-excerpt: This page summarises some of the benefits of segregated witness that go beyond simply increasing the capacity of the block chain.
+excerpt: This page summarises some of the benefits of segregated witness.
 ---
 {% include _toc.html %}
 
-The Segregated Witness soft-fork (segwit) includes a wide range of features, many of which are highly technical. This page summarises some of the benefits of those features that go beyond simply increasing the capacity of the block chain.
+The Segregated Witness soft-fork (segwit) includes a wide range of features, many of which are highly technical. This page summarises some of the benefits of those features.
+
+## Table of Contents
+
+* [Malleability Fixes][]
+* [Linear scaling of sighash operations][]
+* [Signing of input values][]
+* [Increased security for multisig via pay-to-script-hash (P2SH)][]
+* [Script versioning][]
+* [Reducing UTXO growth][]
+* [Compact fraud proofs][]
+* [Efficiency gains when not verifying signatures][]
+* [Block capacity/size increase][]
+* [Moving towards a single combined block limit][]
```

Comment:
This is unnecessary, the TOC is rendered automatically. This change also doesnt render properly.